### ISSUE TITLE
Difference Between Sigmoid Function and Softmax Function

### ISSUE THREAD

**Issue by lemmaa** [lemmaa/journal#2]
The below are the tabular differences between Sigmoid and Softmax function.

  | Softmax Function | Sigmoid Function
-- | -- | --
1 | Used for multi-classification in logistic regression model. | Used for binary classification in logistic regression model.
2 | The probabilities sum will be 1 | The probabilities sum need not be 1.
3 | Used in the different layers of neural networks. | Used as activation function while building neural networks.
4 | The high value will have the higher probability than other values. | The high value will have the high probability but not the higher probability.

- Softmax: Used for the multi-classification task.
- Sigmoid: Used for the binary classification task.

http://dataaspirant.com/2017/03/07/difference-between-softmax-function-and-sigmoid-function/
